### ISSUE TITLE
Wrong ODRL context link

### ISSUE THREAD

**Issue by wolf4ood** [eclipse-dataspace-protocol-base/DataspaceProtocol#71]
# Bug Report

## Describe the Bug

Currently the `odrl` context imported points to `http://www.w3.org/dspace-odrl/odrl.jsonld` which does not exists.

https://github.com/eclipse-dataspace-protocol-base/DataspaceProtocol/blob/main/artifacts/src/main/resources/context/dspace.jsonld#L46

Since the `odrl` profiled context is applied in `dspace` protocol, it would probably make sense to have it hosted under `https://w3id.org/dspace/..`

**Comment by jimmarino** [comment]
@arnoweiss

**Comment by arnoweiss** [comment]
[As mentioned](https://github.com/eclipse-dataspace-protocol-base/DataspaceProtocol/discussions/74), I think the updated schemas and - well - the calendar justify a new release. Apart from that, the IRI structure works for me.

**Comment by jimmarino** [comment]
Should we call it 2024/2 since we should have a release this year, or 2025/1? Maybe the latter?

**Comment by arnoweiss** [comment]
I'd argue for the latter. If this is consensus, I'll create issues for IRIs in artifacts, spec-texts and redirects.

**Comment by mkollenstart** [comment]
> Let's use `http://www.w3.org/dspace/2024/1/odrl.jsonld`. What do others think?

I agree to put it in `dspace`, perhaps it might be a good idea to rename the file to `odrl-profile.jsonld`. To clearly indicate that it is a profile and not just a copy of the original odrl context.

> I'd argue for the latter. If this is consensus, I'll create issues for IRIs in artifacts, spec-texts and redirects.

I would also be in favor of `2025/1`

**Comment by jimmarino** [comment]
OK how about the following?

`http://www.w3.org/dspace/2025/1/odrl-profile.jsonld`

**Comment by arnoweiss** [comment]
Only seeing this now. The ODRL profile is currently available at [`https://w3id.org/dspace/2025/1/odrl.jsonld`](https://w3id.org/dspace/2025/1/odrl.jsonld). [`https://w3id.org/dspace/2025/1/context.jsonld`](https://w3id.org/dspace/2025/1/context.jsonld) is attempting to import `http://www.w3.org/dspace-odrl/odrl.jsonld`. Thus, this will require another PR to w3id AND the artifacts here.
